source/includes/changelogs/releases/100.7.3.rst

```diff
@@ -0,0 +1,22 @@
+.. _100.7.3-changelog:
+
+100.7.3 Changelog
+-----------------
+
+*Released 2023-06-26*
+
+This release fixes an issue introduced in 100.7.1 where specifying 
+``mongodump --db=confg`` will not dump the entire config collection.
```

Comment:
[suggestion]

Maybe "would not" instead of "will not" ?

Reply:
Fixed